dev-lisp/sbcl/sbcl-2.2.4.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="git://github.com/sbcl/sbcl.git"
```

Comment:
github has disabled the unauthenticated git:// protocol. This will need to be updated.

Reply:
updated to use https protocol instead